frontend/src/components/RegisterForm/Header/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export const Header = () => {
+    return (
+        <div className="flex flex-col gap-3">
+            <h1 className="font-semibold text-3xl text-center">
+                Create account
+            </h1>
+            <div className="flex flex-row gap-2">
```

Comment:
Suggest removing caps lock, and adding a `justify-center` to the flex div to have the text and link centered.
![image](https://user-images.githubusercontent.com/105944267/213957910-208d9253-0861-45e1-a9f5-ce288226aa1b.png)